tests/test_text2fhir.py

```diff
@@ -355,3 +355,14 @@ def test_nlp_fhir(self):
             'Observation',
             'Procedure',
         }, {type(x).__name__ for x in fhir})
+
+    def test_nlp_fhir_publication_example(self):
+        file_txt = PathResource.TEXT2FHIR_TEXT.value
+        file_json = PathResource.TEXT2FHIR_JSON.value
+        text = filesystem.read_text(file_txt)
+
+        dict_json = list()
+        for res in text2fhir.text2fhir(text):
+            dict_json.append(res.as_json())
+
+        filesystem.write_json(file_json, dict_json)
```

Comment:
This looks like code that you used to generate the resulting json from the text input. But we shouldn't be writing to a git-managed file inside a unit test.

You could switch this test to *read* `text2fhir.json` and compare that to `dict_json`. And take out the `write_json` line. Then you'd be testing that we never regress from this point on.

Something like (warning: I did not test this):

```
        text = LoadResource.TEXT2FHIR_TEXT.value
        expected_json = LoadResource.TEXT2FHIR_JSON.value
        ctakes_json = [x.as_json() for x in text2fhir.text2fhir(text)]

        self.assertDictEqual(ctakes_json, expected_json)
```